scripts/manage_requirements.py

```diff
@@ -186,9 +186,18 @@ def run(self):
         ambiguities = PinnedRequirements(req for req in overlap if len(req.versions) > 1)
         for req in ambiguities:
             build_req = build_reqs[req]
-            log.warning('Pinning transitive runtime requirement %s to %s, the '
-                        'version resolved at build time.', req, build_req.versions)
-            runtime_reqs[req] = build_req
+            # We can't resolve these ambiguities automatically because different
```

Comment:
Add a paragraph about this to https://github.com/DataBiosphere/azul#11-managing-dependencies. It should include a definition of what an ambiguity is, this text and an example of the exception and the corresponding resolution.